wit/deps/spin@unversioned/mqtt.wit

```diff
@@ -0,0 +1,6 @@
+interface mqtt {
+  use mqtt-types.{payload, error, qos};
+
+  // Publish an MQTT message to the specified topic and return an error, if any.
+  publish: func(address: string, username: string, password: string, keepaliveinterval: u64, topic: string, payload: payload, qos: qos) -> result<_, error>;   
```

Comment:
Suggest `keepalive-interval` or `keep-alive-interval` instead of all the words run together.